### PR TITLE
Ea/27 29

### PR DESCRIPTION
**Pull request by eamouzou**
- Create test, route, controller action, and view for profile order page, and order show page

### CODE REVIEW

File: app/views/orders/show.html.erb [hunk 1]
```diff
@@ -31,19 +35,24 @@
...
 </table>
 
 <section id="grandtotal">
+  <p>Total Item Quantity: <%= @order.total_item_quantity %> </p>
   <p>Total: <%=number_to_currency(@order.grandtotal)%></p>
 </section>
 <section id="datecreated">
-  <p> <%= @order.created_at%></p>
+  <p>Order Made On: <%= @order.created_at%></p>
+  <p>Order Last Updated On: <%= @order.updated_at %></p>
```
Comment (eamouzou) [inline app/views/orders/show.html.erb:57]: Added order specific information to order show page without failing earlier show page tests. Image is sized to fit in the table.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -7,3 +7,4 @@ Zip Code: <%= @user.zip %></br>
 Email: <%= @user.email %></br>
 <%= link_to 'Edit Profile', "/profile/#{@user.id}/edit" %>
 <%= link_to 'Edit Password', "/profile/#{@user.id}/edit_password" %>
+<%= link_to 'My Orders', "/profile/orders" %>
```
Comment (eamouzou) [inline app/views/users/show.html.erb:10]: add link to profile's orders page

File: config/routes.rb [hunk 3]
```diff
@@ -45,6 +45,7 @@
 
   get "/profile/:profile_id/edit_password", to: 'users#edit_password'
   patch "/profile/:profile_id", to: 'users#update_password'
+  get '/profile/orders/:id', to: 'orders#show'
```
Comment (eamouzou) [inline config/routes.rb:48]: route for profile show page

File: spec/features/orders/show_spec.rb [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      expect(page).to have_content(@paper.name)
+      expect(page).to have_css("img[src='#{@paper.image}']")
+      expect(page).to have_content(@paper_order.quantity)
+      expect(page).to have_content(@paper.price)
+      expect(page).to have_content(@paper_order.subtotal)
+    end  
+  end
+
+
+end
```
Comment (eamouzou) [inline spec/features/orders/show_spec.rb:60]: add test for orders show page

File: spec/features/orders/index_spec.rb [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      expect(page).to have_link(@user_order.id)
+      expect(page).to have_content("Order ID: #{@user_order.id}")
+      expect(page).to have_content(@user_order.created_at)
+      expect(page).to have_content(@user_order.updated_at)
+      expect(page).to have_content(@user_order.status)
+      expect(page).to have_content("Total Item Quantity: 4")
+      expect(page).to have_content("Grand Total of All Items: $142.00")
+    end
+  end
+end
```
Comment (eamouzou) [inline spec/features/orders/index_spec.rb:63]: add test for orders index page

**Review by agallant121 — APPROVED** [review]

**Pull request merged**